server/models/userModel.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose';
+import bcrypt from 'bcryptjs';
+
+const UserSchema = new mongoose.Schema({
+  name: {
+    type: String,
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  createdAt: {
```

Comment:
you can add the timestamp option in mongoose https://mongoosejs.com/docs/guide.html#timestamps, then you dont have to set default, also you can also get the updated at field.